packages/zowe-explorer/src/utils/AuthUtils.ts

```diff
@@ -177,8 +177,16 @@ export class AuthUtils {
             return;
         }
         sessionNode.setProfileToChoice(profile);
-        const session = getCommonApi(profile).getSession();
-        sessionNode.setSessionToChoice(session);
+        try {
+            const commonApi = getCommonApi(profile);
+            sessionNode.setSessionToChoice(commonApi.getSession());
+        } catch (err) {
+            if (err instanceof Error) {
+                // API is not yet registered, or building the session failed for this profile
+                ZoweLogger.error(`Error syncing session for ${profileName}: ${err.message}`);
+            }
```

Comment:
I know I've mentioned this before... 😋 
I just feel strange every time I see an `instanceOf Error` check in a catch block 😅 
No need to do this though 😋 

```suggestion
            // API is not yet registered, or building the session failed for this profile
            ZoweLogger.error(`Error syncing session for ${profileName}: ${err}`);
```